packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -626,10 +628,12 @@ const extensions =
                                   test: /\.ts?$/,
                                   use: {
                                       loader: findDepInStack('ts-loader'),
-                                      // No nothing if tsconfig.json is declared in the extension.
-                                      // Typescript will resolve to it magically.
-                                      // If no tsconfig.json is found, use the default options
-                                      ...(tsConfigFound ? {} : {compilerOptions: defaultTsConfig})
+                                      options: {
+                                          // If tsconfig is not found, ignore tsconfig.json and rely on
+                                          // using the default compilerOptions
+                                          happyPackMode: !tsConfigFound,
```

Comment:
Please ad a comment in the form of a TODO here that we want to NOT use this argument in the future, but it was required to get your pr working